src/main/java/org/jvnet/hudson/test/TestPluginManager.java

```diff
@@ -63,11 +66,24 @@ private TestPluginManager() throws IOException {
     @Override
     protected Collection<String> loadBundledPlugins() throws Exception {
         Set<String> names = new HashSet<String>();
+        
+        names.addAll(loadBundledPlugins(new File(WarExploder.getExplodedDir(), "WEB-INF/plugins")));
+        loadBundledPlugins(new File(WarExploder.getExplodedDir(), "WEB-INF/detached-plugins"));
+                
+        return names;
+    }
+
+    private Set<String> loadBundledPlugins(File fromDir) throws IOException, URISyntaxException {
+        if (!fromDir.exists()) {
+            LOGGER.log(Level.INFO, "No plugins loaded from " + fromDir + ". Directory doesn't exist.");
```

Comment:
Maybe shut this up for the common case that you are running against 1.0 and `detached-plugins` is not _supposed_ to exist.